### DISCUSSION TITLE
Trappings - Containers - Ability to allocate items to containers so ENC is not counted

### DISCUSSION THREAD

**Discussion by MogliWorld** [fmasa/wfrp-master#57] [Ideas]
Trappings - Containers - Ability to allocate items to containers so ENC is not counted

A pouch weighs nothing but holds 1
A sling bag weighs 1 but when worn is 0, but holds 2.
A Backpack weighs 2 but when worn is 1 but holds 4.

The when worn works, but could you add the ability to allocate items to the containers, so their ENC is not counted?  Not sure how you achieve this without adding containers to compendium.  Obviously this can be done in descriptions.

Mo

**Comment by fmasa** [comment]
Hi, yes this is something that is planned as part of https://github.com/fmasa/wfrp-master/issues/9, from mechanics perspective, this already works, it's just not available in the UI, since I'm still not sure about the best way to do this.

**Comment by MogliWorld** [comment]
That's good to hear.

A possible solution would be to have a container ENC total with a container
checkbox like worn. So if an item selects the checkbox its considered to be
in the containers. Description could be used to specify which, or an extra
Description field added to specify which.

**Reply by fmasa** [reply]
I went with a different approach, but would love the feedback. [Version 3.7.0](https://github.com/fmasa/wfrp-master/releases/tag/3.7.0) should be in Google Play in a few hours, could you try the new Container UI and let me know what you think?

**Comment by MogliWorld** [comment]
Sorry,  only just spotted your email.

I really like your approach, I was playing with it last night.

Ranged weapons are showing as Melee weapons on the descriptions though,
that's the only issue I saw from the little play I had.  I moved trappings
about to test it out, they add and remove perfectly well.

Your improvements are really good, I only use the player part and I'm the
only one at our group that uses it. The others and GM are still paper
fanatics, not managed to convince them yet even though it works so well on
a phone.

**Reply by fmasa** [reply]
> Ranged weapons are showing as Melee weapons on the descriptions though,
that's the only issue I saw from the little play I had.

Good catch, fixed in #68 and released as part of 3.7.1 :slightly_smiling_face:

**Comment by MogliWorld** [comment]
Create a new Melee Weapon or ranged weapon then try to view it. Crashes the
app.

Existing weapons work as expected.

**Reply by fmasa** [reply]
Ah, yes. Previously leading + in damage was not supported. I allowed this format in 3.7.0, but not completely. I just released 3.7.2, which fixes that.

**Reply by fmasa** [reply]
Thanks for report :)

**Comment by MogliWorld** [comment]
Ahh, that's what I was doing.

I'll let you know if I see anything else weird.

Merry Christmas. 🎄
